test/utils/P256.t.sol

```diff
@@ -133,6 +144,7 @@ contract P256Test is Test {
         string calldata message
     ) public {
         (, uint256 key) = makeAddrAndKey(signer);
+        key = bound(key, 1, _N - 1);
```

Comment:
maybe instead just get the uint256 from fuzzing and bound it, without needing `makeAddrAndKey`